src/lib/kimyi.rs

```diff
@@ -655,8 +657,7 @@ E: 'OPEN_BRACKET' E 'CLOSE_BRACKET'
         assert_eq!(errs[0].repairs().len(), 1);
         check_repairs(&grm,
                       errs[0].repairs(),
-                      &vec!["InsertTerm \"A\", InsertTerm \"CLOSE_BRACKET\", InsertTerm \"CLOSE_BRACKET\"",
-                            "InsertTerm \"B\", InsertTerm \"CLOSE_BRACKET\", InsertTerm \"CLOSE_BRACKET\""]);
+                      &vec!["InsertNonterm \"E\", InsertTerm \"CLOSE_BRACKET\", InsertTerm \"CLOSE_BRACKET\""]);
```

Comment:
Why does renaming `A` to `E` needs removing `B`?

Reply:
That's not really what's going on here: it's the change from `InsertTerm` to `InsertTerm`. That's a knock-on effect of changing the hashing algorithm. Put another way, the hashing algorithm affects which repairs are considered equivalent to each other; changing it implies that different repairs may be found (as in the case here).

Reply:
Ah, missed the `InsertNonterm`.